### PR TITLE
Adds grenadine.

### PR DESCRIPTION
**Pull request by Hyacinth-OR**
:cl: Hyacinth
add: Grenadine is now available in the bar!
tweak: Tequila sunrises now require grenadine to make!
/:cl:

Grenadine is an essential part of a barkeeper's toolkit. In addition, being a huge fan of tequila sunrises in real life, I've always been disappointed that the code passes off a tequila screwdriver as a tequila sunrise, so I've remedied that, and edited the tequila sunrise sprite to reflect this.

I'm looking forward to hearing any of your questions comments or concerns :D!

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by europax** [comment]
From #37589 here - possible merge conflict incoming

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -361,6 +361,16 @@
 	list_reagents = list("limejuice" = 100)
 	foodtype = FRUIT
 
+/obj/item/reagent_containers/food/drinks/bottle/grenadine
+	name = "Jester Grenadine"
+	desc = "Contains 0% real cherries!"
+	icon_state = "grenadine"
+	lefthand_file = 'icons/mob/inhands/equipment/kitchen_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/kitchen_righthand.dmi'
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: There's no item_state defined for this, so changing these inhand files won't do anything

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 2]
```diff
@@ -632,6 +632,7 @@
 	required_reagents = list("rice" = 10)
 	required_catalysts = list("enzyme" = 5)
 	mix_message = "The rice grains ferment into a clear, sweet-smelling liquid."
+<<<<<<< .mine
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: You messed up the merge

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by improvedname** [comment]
Drink bloat that one thing that might ever get used in a super rare rp moment on tg

**Comment by epochayur** [comment]
do you get paid to make pointless, obnoxious comments, or do you just really enjoy it that much

**Comment by ExcessiveUseOfCobblestone** [comment]
I plan on giving these functionality after merge so shush zootopia

**Review by WJohn — DISMISSED** [review]
Not a super unique sprite, but I do like the little rainbow going on in the center and the bright red is cheerful. Fine by me.

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 3]
```diff
@@ -633,6 +633,37 @@
 	required_catalysts = list("enzyme" = 5)
 	mix_message = "The rice grains ferment into a clear, sweet-smelling liquid."
 
+
+/datum/chemical_reaction/alexander
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: Dupe 

https://github.com/tgstation/tgstation/pull/37598/files#diff-96dc0c287671d3dd23c39f617ecf5ad6R667

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**